DOCUMENTATION.md

```diff
@@ -49,66 +49,66 @@ client.close()
 The callback functions are functions which are executed when a trigger event happens.
 
 #### set_uplink_callback
-Set a callback function, to be called when an uplink message is received. It's possible to add more than one callback function.
+Add a callback function, to be called when an uplink message is received.
 ```python
 client.set_uplink_callback(uplink_callback)
 ```
 
 ##### uplink_callback
 The callback function must be declared in the script following this structure:
 * `uplink_callback(msg, client)`
-  * `msg`: **JSON**  the message received by the client
-  * `client`: **object**  the client from which the callback is executed are calling
+  * `msg`: **object**  the message received by the client.
```

Comment:
Please clarify whether it is an `Object` (and in that class, the class of this object) or if it is a `dict`